Error in ConstitutiveModelsApplication when compiling

**Description**
I just compilied for for the first time in a while and it is giving me an error with the constitutive models application. Did I miss something that has changed in the past couple months? Anyone else have this issue?
```
~/opt/kratos/applications/ConstitutiveModelsApplication/custom_processes/non_local_plasticity_process.cpp: In member function ‘void Kratos::NonLocalPlasticityProcess::PerformGaussPointSearch(std::vector<Kratos::NonLocalPlasticityProcess::GaussPoint>&, double)’:
~/opt/kratos/applications/ConstitutiveModelsApplication/custom_processes/non_local_plasticity_process.cpp:195:14: error: ‘class Kratos::Element’ has no member named ‘GetValueOnIntegrationPoints’; did you mean ‘SetValuesOnIntegrationPoints’?
  195 |          ie->GetValueOnIntegrationPoints(CONSTITUTIVE_LAW, ConstitutiveLawVector, rCurrentProcessInfo);
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |              SetValuesOnIntegrationPoints
```

**Environment**
- OS: Kubuntu 20.10
- Branch: master
- Python 3.8.6

--- a/applications/ConstitutiveModelsApplication/custom_processes/non_local_plasticity_process.cpp
+++ b/applications/ConstitutiveModelsApplication/custom_processes/non_local_plasticity_process.cpp
@@ -192,7 +192,7 @@ namespace Kratos
 
 
          std::vector<ConstitutiveLaw::Pointer> ConstitutiveLawVector(numberOfGP);
-         ie->GetValueOnIntegrationPoints(CONSTITUTIVE_LAW, ConstitutiveLawVector, rCurrentProcessInfo);
+         ie->CalculateOnIntegrationPoints(CONSTITUTIVE_LAW, ConstitutiveLawVector, rCurrentProcessInfo);
 
 
 

